### ISSUE TITLE
stable hlo xor binary

### ISSUE THREAD

**Issue by AleksKnezevic** [tenstorrent/tt-mlir#1055]
Add support for stable hlo xor binary, and add tests to tt-xla and tt-torch.

**Comment by kmitrovicTT** [comment]
Blocked by https://github.com/tenstorrent/tt-metal/issues/13582

**Comment by skhatriTT** [comment]
@AleksKnezevic looks like we are unblocked. Is this issue resolved?

**Comment by AleksKnezevic** [comment]
@kmitrovicTT ?

**Comment by kmitrovicTT** [comment]
Yes, sorry, didn't catch this one. Tests added long time ago to tt-xla. Closing.
